distribution/lib/Standard/Table/0.0.0-dev/src/Io/File_Format.enso

```diff
@@ -112,3 +118,15 @@ type Delimited
 ## A setting to infer the default behaviour of some option.
 type Infer
 
+## Read the file to a `Table` from an Excel file
+type Excel
+    type Excel (section:Excel_Section=Excel_Module.Sheet_Names) (xls_format:(True|False|Infer)=Infer)
```

Comment:
IIRC the documentation that will show up in the IDE is taken (currently, not sure if going to change?) from this nested definition, so it would be good to fill it up, since this type is pretty important and the user may seek explanation. It would also be good to add an `Arguments:` section here (and to most public functions, maybe apart from ones that have 1-2 arguments and it is completely obvious what they mean). Here for example, I'm not sure what the `xls_format` flag means. What does it means if it is `False`?
```suggestion
## Read the file to a `Table` from an Excel file
type Excel
    ## [Some doc here]
    type Excel (section:Excel_Section=Excel_Module.Sheet_Names) (xls_format:(True|False|Infer)=Infer)
```